soroban-env-host/src/lib.rs

```diff
@@ -46,8 +46,6 @@ mod test;
 
 #[cfg(feature = "testutils")]
 pub use host::ContractFunctionSet;
-#[cfg(feature = "testutils")]
-pub use host::LedgerInfo;
-pub use host::{Host, HostError};
+pub use host::{Host, HostError, LedgerInfo, metered_map::MeteredOrdMap};
```

Comment:
LedgerInfo needs to be in testutils I think?

Reply:
As far as I know it needs to be always available, since it will be populated on production networks.